feature/mypage/src/main/java/com/umcspot/spot/mypage/main/MyPageViewModel.kt

```
@@ -0,0 +1,109 @@
+                )
+            } else {
+                @Suppress("DEPRECATION")
+                pm.getPackageInfo(pkgName, 0)
+            }
+
+            val versionName = packageInfo.versionName ?: "unknown"
+
+            _uiState.update {
+                it.copy(appVersion = UiState.Success(versionName))
+            }
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)